engine/src/flutter/shell/platform/windows/client_wrapper/include/flutter/dart_project.h

```diff
@@ -90,6 +90,15 @@ class DartProject {
   // Defaults to NoPreference.
   GpuPreference gpu_preference() const { return gpu_preference_; }
 
+  // Sets whether the UI isolate should be running on the platform thread.
+  void set_merged_platform_ui_thread(bool merged_platform_ui_thread) {
+    merged_platform_ui_thread_ = merged_platform_ui_thread;
+  }
+
+  // Returns whether the UI isolate should be running on the platform thread.
+  // Defaults to false.
+  bool merged_platform_ui_thread() const { return merged_platform_ui_thread_; }
```

Comment:
I like the comment in: https://github.com/flutter/flutter/pull/162684

What do you think of something like this?

```suggestion
  // Sets whether the UI isolate should run on the platform thread.
  // In a future release, this setting will become a no-op when
  // Flutter Windows requires merged platform and UI threads.
  void set_merged_platform_ui_thread(bool merged_platform_ui_thread) {
    merged_platform_ui_thread_ = merged_platform_ui_thread;
  }

  // Returns whether the UI isolate should run on the platform thread.
  // Defaults to false. In a future release, this setting will default
  // to true.
  /// this setting will become a no-op when the Skia backend is fully removed.
  bool merged_platform_ui_thread() const { return merged_platform_ui_thread_; }
```